.pre-commit-config.yaml

```diff
@@ -0,0 +1,6 @@
+repos:
+  - repo: https://github.com/macisamuele/language-formatters-pre-commit-hooks
+    rev: 42672673f0bed83269d4465ceac63d24ca9c401a
+    hooks:
+      - id: pretty-format-java
```

Comment:
@Googleton will it change indentation too? I used a few `formatter:off` hints to tell Intellij not to touch a couple of places for readability reasons. Will it take it into account?
Is it possible to start by automatic unused import removal?